plugins/clipboard/plugin.js

```diff
@@ -431,6 +454,10 @@
 					editor.openDialog( 'paste', evt.data );
 				}, 0 );
 			} );
+		},
+
+		addNotificationFileMatcher: function( matcher ) {
```

Comment:
There is an inconsistency between the method and property names. Maybe we should rename this method?

```suggestion
		addSupportedFileMatcher: function( matcher ) {
```

Additionally, this method lacks proper API docs.

Reply:
I've moved it to `CKEDITOR.plugins.clipboard` namespace, otherwise, AFAIK docs are not properly generated for plugin instances.